src/theory/quantifiers/quantifiers_rewriter.h

```diff
@@ -233,10 +234,20 @@ class QuantifiersRewriter : public TheoryRewriter
                              std::vector<Node>& activeArgs,
                              Node n,
                              Node ipl);
-  Node computeProcessTerms2(Node body,
+  /**
+   *
+   * It may introduce new conditions C into new_conds. It returns a node retBody
+   * such that q of the form
+   *   forall args. body
+   * is equivalent to:
+   *   forall args. ( C V retBody )
+   */
+  Node computeProcessTerms2(const Node& q,
```

Comment:
Any reason why we use `const Node&` instead of `TNode` here?

Reply:
I feel we have still not converged on which one is best, I had been using `const Node&` in other contexts.